README.md

```diff
@@ -128,6 +128,7 @@ Index of available Reusable Workflows in this repository.
 | Name           | Description                                                                                                       | Docs                                          |
 | -------------- | ----------------------------------------------------------------------------------------------------------------- | --------------------------------------------- |
 | `lint-actions` | Use [Zizmor](https://woodruffw.github.io/zizmor/) to run static analysis checks on Github Actions workflow files. | [:link:](.github/workflows/lint-actions.yaml) |
+| `publish-r2`   | Use rclone to publish static content to Cloudflare's R2                                                           | [:link:](.github/workflows/publish-r2.yaml)   |
```

Comment:
Can you also add a link in the top-level TOC on line 11? I don't think I like this duplication I've created here (or maybe I'd just like to automate sync'ing them) but for now that's how it's laid out :upside_down_face: (I only have myself to blame)

Reply:
Done.